src/vendor/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
This is just an example, right?  If so I think this and the other example files should be moved to examples/vendor.